### ISSUE TITLE
[Help wanted] "dbee" command not found

### ISSUE THREAD

**Issue by TroySigX** [kndndrj/nvim-dbee#73]
When I try to open the UI, an error occurs:
```
DbeeGetCurrentConnection Vim:E475: Invalid value for argument cmd: 'dbee' is not executable
stack traceback:
^I[C]: at 0x562fee0b0130
^I[C]: in function 'pcall'
^I...ocal/share/nvim/lazy/nvim-dbee/lua/dbee/handler/init.lua:160: in function 'get_current_connection'
^I...al/share/nvim/lazy/nvim-dbee/lua/dbee/ui/drawer/init.lua:62: in function 'new'
^I....local/share/nvim/lazy/nvim-dbee/lua/dbee/entry/init.lua:93: in function 'setup_ui'
^I....local/share/nvim/lazy/nvim-dbee/lua/dbee/entry/init.lua:135: in function 'open_ui'
^I....local/share/nvim/lazy/nvim-dbee/lua/dbee/entry/init.lua:130: in function 'toggle_ui'
^I...e/troysigx/.local/share/nvim/lazy/nvim-dbee/lua/dbee.lua:24: in function 'toggle'
^I[string ":lua"]:1: in main chunk
```

Here is my config using `lazy.nvim`:
```lua
{
    'kndndrj/nvim-dbee',
    dependencies = {
      'MunifTanjim/nui.nvim',
    },
    build = function()
      -- Install tries to automatically detect the install method.
      -- if it fails, try calling it with one of these parameters:
      --    "curl", "wget", "bitsadmin", "go"
      require('dbee').install()
    end,
    config = function()
      require('dbee').setup(--[[optional config]])
    end,
  }
```

**Comment by kndndrj** [comment]
can you paste the output of `:checkhealth dbee`

**Comment by TroySigX** [comment]
Here's the output:
```
==============================================================================
dbee: require("dbee.health").check()

DBee report ~
- ERROR Binary not executable: /home/troysigx/.local/share/nvim/dbee/bin/dbee.
```

**Comment by kndndrj** [comment]
try running `require"dbee".install()` and don't close vim until you see the success message (it's a ~25mb download).

**Comment by TroySigX** [comment]
It seems that because of this issue #72, it doesn't download at all when I first install the plugin

**Comment by kndndrj** [comment]
Fixed with #98
